### PR TITLE
Support adding/deleting reviewers and labels

### PR DESCRIPTION
**Pull request by RMacfarlane**
Builds on top of #951 
https://github.com/Microsoft/vscode-pull-request-github/issues/412

Adding uses the quickpick:
add_reviewer

Icon to remove appears on hover when the view is shown as a sidebar:
delete_reviewer

When the window is less than 900px wide, content looks like, with delete icons always shown:
screen shot 2019-02-15 at 4 08 07 pm

### CODE REVIEW

**Comment by rebornix** [comment]
Run into following issues

* Open PR description of https://github.com/auchenberg/pullrequest-demo/pull/45
* Try to add a label
* Labels already assigned are not shown on the list
* Add one (say `question`)
* `question` is added successfully
* Click the + icon again, `question` is still on the list.


* Open PR description of https://github.com/auchenberg/pullrequest-demo/pull/45
* Split the editor
* Click the + icon to add a label, nothing happens (I think the quick pick shows up and dismisses again)
* Repeat the steps again and this time try to remove a label, it works as expected

File: src/github/pullRequestManager.ts [hunk 1]
```diff
@@ -1249,6 +1276,51 @@ export class PullRequestManager {
...
+			owner: remote.owner,
+			repo: remote.repositoryName,
+			number: pullRequest.prNumber,
+			reviewers: [ reviewer ]
+		});
+	}
+
+	async addLabels(pullRequest: PullRequestModel, labels: string[]): Promise<void> {
+		const { octokit, remote } = await pullRequest.githubRepository.ensure();
+		octokit.issues.addLabels({
```
Comment (rebornix) [inline src/github/pullRequestManager.ts]: do we want to `await` this call?

File: src/github/pullRequestManager.ts [hunk 2]
```diff
@@ -1249,6 +1276,51 @@ export class PullRequestManager {
...
+			owner: remote.owner,
+			repo: remote.repositoryName,
+			number: pullRequest.prNumber,
+			labels
+		});
+	}
+
+	async removeLabel(pullRequest: PullRequestModel, label: string): Promise<void> {
+		const { octokit, remote } = await pullRequest.githubRepository.ensure();
+		octokit.issues.removeLabel({
```
Comment (rebornix) [inline src/github/pullRequestManager.ts]: same here.

File: src/github/pullRequestManager.ts (hunk 1 above)
Comment (RMacfarlane) [reply src/github/pullRequestManager.ts]: yeah, nice catch

**Comment by RMacfarlane** [comment]
Fixed the label filtering problem, the quickpick issue seems to be a problem with the webview getting refocused, so I think it might be an upstream issue

**Pull request merged**